apps/aehttp/test/aehttp_integration_SUITE.erl

```diff
@@ -6003,6 +6043,8 @@ data_code_to_reason([1006])      -> <<"broken_encoding: contracts">>;
 data_code_to_reason([1007])      -> <<"contract_init_failed">>;
 data_code_to_reason([1008])      -> <<"not_a_number">>;
 data_code_to_reason([1005,1006]) -> <<"broken_encoding: accounts, contracts">>;
```

Comment:
When is this exercised?

Reply:
On line 4750

Reply:
I see this is test code only - sorry for the noise. Thanks.

Reply:
https://github.com/aeternity/aeternity/blob/172733506a526b688eea795090b1158694c9581e/apps/aehttp/test/aehttp_integration_SUITE.erl#L4750